lib/portlayer/exec/handle.go

```diff
@@ -244,9 +251,8 @@ func Create(ctx context.Context, sess *session.Session, config *ContainerCreateC
 	}
 
 	specconfig := &spec.VirtualMachineConfigSpecConfig{
-		// FIXME: hardcoded values
-		NumCPUs:  2,
-		MemoryMB: 2048,
+		NumCPUs:  int32(config.Resources.NumCPUs),
```

Comment:
why we are defining this as int64 in the struct and then casting it to int32, can't we start with int32? Don't think we will overflow but just in case :)